api/middleware/notesMiddleware.js

```diff
@@ -19,15 +19,23 @@ const checkNoteExists = async (req, res, next) => {
 };
 
 const checkBodyIsComplete = (req, res, next) => {
+  const {
+    content_type,
+    content,
+    level,
+    visible_to_admin,
+    visible_to_mentor,
+    mentor_id,
+    mentee_id,
+  } = req.body;
```

Comment:
I love this destructuring from request body, it makes your code look simpler and easy to read.